### PR TITLE
docs: Improve start mvp servers docs.

### PR DESCRIPTION
**Pull request by yanguoyu**
> Follow the [Guide](../../../README.md) to build libraries developed in other workspaces.

I guess this guide may not clearly describe how to build the dependence for `mvp-dapp`.
So I write docs that how to start mvp server step by step.

### CODE REVIEW

File: packages/samples/mvp-dapp/README.md [hunk 1]
```diff
@@ -127,12 +123,39 @@ Follow https://github.com/ckb-js/kuai/issues/306 to deploy your own contract. Or
...
+```sh
+$ git clone https://github.com/ckb-js/kuai.git
+```
+
+#### Build dependencies
 
 ```sh
+$ npm i
 $ npm run build
 ```
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/README.md:139]:
```suggestion
```sh
$ cd kuai
$ npm i
$ npm run build
```
```

File: packages/samples/mvp-dapp/README.md [hunk 2]
```diff
@@ -127,12 +123,39 @@ Follow https://github.com/ckb-js/kuai/issues/306 to deploy your own contract. Or
...
 
+#### Run the mvp dapp
+
+- run dev
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/README.md:143]:
```suggestion
- run the mvp dapp in develop mode
```

File: packages/samples/mvp-dapp/README.md [hunk 3]
```diff
@@ -127,12 +123,39 @@ Follow https://github.com/ckb-js/kuai/issues/306 to deploy your own contract. Or
...
+
+```sh
+$ cd ./packages/samples/mvp-dapp
+$ npm run dev
+```
+
+- run prod
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/README.md:150]:
```suggestion
- run the mvp dapp in production mode
```

File: packages/samples/mvp-dapp/README.md [hunk 4]
```diff
@@ -127,12 +123,39 @@ Follow https://github.com/ckb-js/kuai/issues/306 to deploy your own contract. Or
...
-### Build the mvp dapp
+### Start server
+
+#### Clone this project locally
+
+```sh
+$ git clone https://github.com/ckb-js/kuai.git
+```
+
+#### Build dependencies
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/README.md:134]:
```suggestion
#### Install dependencies and build kuai modules locally
```

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
Section of `Start Kuai MVP DApp UI` can be appended by adding its link https://github.com/Magickbase/kuai-mvp-dapp-ui

**Comment by Keith-CY** [comment]
> Section of `Start Kuai MVP DApp UI` can be appended by adding its link [Magickbase/kuai-mvp-dapp-ui](https://github.com/Magickbase/kuai-mvp-dapp-ui)

Will the link of front-end be added in this PR?

BTW, please have a review @Daryl-L @PainterPuppets

**Review by Daryl-L — APPROVED** [review]

**Pull request merged**